qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -257,6 +258,40 @@ public void acceptsConstructorParametersNamedJustLikeFields()
     }
 
     /**
+     * CheckstyleValidator allows local variables and catch parameters with
+     * names matching {@code ^[a-z]{3,12}$} pattern.
+     * Additionally, catch parameters can use name {@code ex}.
+     * @throws Exception In case of error
+     */
+    @Test
+    public void allowsOnlyProperlyNamedLocalVariables() throws Exception {
```

Comment:
@mkordas it would be also good to check that number of names check violations is equal to the number you are checking. Basically to make sure that all the names in `valid()` method are passing this check.